open_academy/models/session.py

```diff
@@ -1,5 +1,6 @@
 from odoo import models, fields, api
 from odoo.exceptions import ValidationError
+from odoo.tools.translate import _
```

Comment:
remove this line, odoo has the translation module
```suggestion
from odoo import models, fields, api, _
```